database/index.js

```diff
@@ -6,12 +6,64 @@ const pool = new Pool({
 });
 
 module.exports = {
-  getOne: (callback) => {
-    pool.query('SELECT * FROM reviews ORDER BY id ASC', (error, datas) => {
+  getAll: (id, callback) => {
+    pool.query(`SELECT * FROM reviews WHERE roomId = ${id} ORDER BY id ASC`, (error, datas) => {
       if (error) {
         throw error;
       }
       callback(null, datas.rows);
     });
   },
+
+  getOne: (ids, callback) => {
+    // console.log(ids);
+    //{ roomId: '210329', reviewId: '746820' }
+    pool.query(`SELECT * FROM reviews WHERE roomId = ${ids.roomId} AND id = ${ids.reviewId} ORDER BY id ASC`, (error, datas) => {
```

Comment:
Just FYI data is already plural so I wouldn't use datas

Reply:
Got it! I am really bad at naming variables... thx!